test/e2e/PISP_Accounts.test.ts

```diff
@@ -0,0 +1,55 @@
+import TestEnv from './TestEnv'
+import axios from 'axios'
+
+describe('GET /accounts/{ID}', (): void => {
+  const requestConfig = {
+    headers: {
+      'FSPIOP-Source': 'pisp',
+      'FSPIOP-Destination': 'dfspa',
```

Comment:
there is a need to pass dfsp via path parameter

Reply:
Ok, incorporated https://github.com/mojaloop/thirdparty-scheme-adapter/pull/61 changes